lio-http-server/examples/HelloWorld.hs

```diff
@@ -1,9 +1,9 @@
 {-# LANGUAGE OverloadedStrings #-}
 import LIO.HTTP.Server
+import LIO.HTTP.Server.Responses
 
 main :: IO ()
 main = server 3000 "127.0.0.1" app
 
 app :: DCApplication
-app _ = do
-  return $ Response status200 [] "Hello World!"
+app _ = return (okHtml "Hello World!")
```

Comment:
can you just create another file? Or add another app. We want both: ability to create responses manually and use the responses API. Adding additional examples, again even if in same file, that exercise other APIs exposed by lib would be great.